Makefile

```diff
@@ -394,13 +395,19 @@ test\#%:
 coverage:
 	grep '^\(mode: .*\)\|\(.*:[0-9]\+\.[0-9]\+,[0-9]\+\.[0-9]\+ [0-9]\+ [0-9]\+\)$$' coverage.out > coverage-bodged.out
 	grep '^\(mode: .*\)\|\(.*:[0-9]\+\.[0-9]\+,[0-9]\+\.[0-9]\+ [0-9]\+ [0-9]\+\)$$' integration.coverage.out > integration.coverage-bodged.out
-	$(GO) run build/gocovmerge.go integration.coverage-bodged.out coverage-bodged.out > coverage.all
+	$(GO) run build/gocoverage.go merge integration.coverage-bodged.out coverage-bodged.out > coverage.all
 
 .PHONY: unit-test-coverage
 unit-test-coverage:
 	@echo "Running unit-test-coverage $(GOTESTFLAGS) -tags '$(TEST_TAGS)'..."
 	@$(GO) test $(GOTESTFLAGS) -timeout=20m -tags='$(TEST_TAGS)' -cover -coverprofile coverage.out $(GO_TEST_PACKAGES) && echo "\n==>\033[32m Ok\033[m\n" || exit 1
 
+.PHONY: test-coverage-check
+test-coverage-check:
+	@$(GO) run build/gocoverage.go check coverage.all \
+		code.gitea.io/gitea/modules/setting=7.1% \
+		code.gitea.io/gitea/modules/hostmatcher=68.6%
```

Comment:
I don't know if it is a good idea to "store" the results in the Makefile.
Anyway here are some more entries for the current code base:
```
code.gitea.io/gitea/modules/activitypub=60.66%
code.gitea.io/gitea/modules/analyze=73.91%
code.gitea.io/gitea/modules/auth=60.29%
code.gitea.io/gitea/modules/avatar=55.20%
code.gitea.io/gitea/modules/base=92.96%
code.gitea.io/gitea/modules/cache=14.60%
code.gitea.io/gitea/modules/charset=56.67%
code.gitea.io/gitea/modules/container=100.00%
code.gitea.io/gitea/modules/context=58.91%
code.gitea.io/gitea/modules/csv=100.00%
code.gitea.io/gitea/modules/doctor=20.87%
code.gitea.io/gitea/modules/emoji=85.11%
code.gitea.io/gitea/modules/eventsource=53.88%
code.gitea.io/gitea/modules/generate=0.00%
code.gitea.io/gitea/modules/git=61.93%
code.gitea.io/gitea/modules/gitgraph=75.77%
code.gitea.io/gitea/modules/graceful=29.29%
code.gitea.io/gitea/modules/hcaptcha=0.00%
code.gitea.io/gitea/modules/highlight=67.14%
code.gitea.io/gitea/modules/hostmatcher=79.34%
code.gitea.io/gitea/modules/html=100.00%
code.gitea.io/gitea/modules/httpcache=78.95%
code.gitea.io/gitea/modules/httplib=67.86%
code.gitea.io/gitea/modules/indexer=39.21%
code.gitea.io/gitea/modules/issue=75.47%
code.gitea.io/gitea/modules/json=43.75%
code.gitea.io/gitea/modules/lfs=70.98%
code.gitea.io/gitea/modules/log=69.41%
code.gitea.io/gitea/modules/markup=53.32%
code.gitea.io/gitea/modules/mcaptcha=0.00%
code.gitea.io/gitea/modules/metrics=0.00%
code.gitea.io/gitea/modules/migration=70.45%
code.gitea.io/gitea/modules/mirror=16.67%
code.gitea.io/gitea/modules/nosql=29.21%
code.gitea.io/gitea/modules/notification=64.94%
code.gitea.io/gitea/modules/options=48.74%
code.gitea.io/gitea/modules/packages=70.63%
code.gitea.io/gitea/modules/paginator=100.00%
code.gitea.io/gitea/modules/password=66.18%
code.gitea.io/gitea/modules/pprof=0.00%
code.gitea.io/gitea/modules/private=13.40%
code.gitea.io/gitea/modules/process=40.28%
code.gitea.io/gitea/modules/proxy=21.43%
code.gitea.io/gitea/modules/proxyprotocol=0.00%
code.gitea.io/gitea/modules/public=53.26%
code.gitea.io/gitea/modules/queue=39.17%
code.gitea.io/gitea/modules/recaptcha=0.00%
code.gitea.io/gitea/modules/references=80.50%
code.gitea.io/gitea/modules/regexplru=65.22%
code.gitea.io/gitea/modules/repository=49.68%
code.gitea.io/gitea/modules/secret=64.00%
code.gitea.io/gitea/modules/session=18.63%
code.gitea.io/gitea/modules/setting=54.19%
code.gitea.io/gitea/modules/sitemap=82.35%
code.gitea.io/gitea/modules/ssh=45.98%
code.gitea.io/gitea/modules/storage=57.14%
code.gitea.io/gitea/modules/structs=54.22%
code.gitea.io/gitea/modules/svg=96.15%
code.gitea.io/gitea/modules/sync=100.00%
code.gitea.io/gitea/modules/system=70.00%
code.gitea.io/gitea/modules/templates=43.01%
code.gitea.io/gitea/modules/test=68.83%
code.gitea.io/gitea/modules/timeutil=80.80%
code.gitea.io/gitea/modules/translation=60.70%
code.gitea.io/gitea/modules/typesniffer=92.86%
code.gitea.io/gitea/modules/updatechecker=0.00%
code.gitea.io/gitea/modules/upload=73.85%
code.gitea.io/gitea/modules/uri=27.78%
code.gitea.io/gitea/modules/user=23.53%
code.gitea.io/gitea/modules/util=72.93%
code.gitea.io/gitea/modules/validation=80.12%
code.gitea.io/gitea/modules/watcher=0.00%
code.gitea.io/gitea/modules/web=76.76%
code.gitea.io/gitea/modules/webhook=13.79%
code.gitea.io/gitea/models/activities=58.48%
code.gitea.io/gitea/models/admin=33.55%
code.gitea.io/gitea/models/asymkey=41.40%
code.gitea.io/gitea/models/auth=46.36%
code.gitea.io/gitea/models/avatars=31.82%
code.gitea.io/gitea/models/db=41.53%
code.gitea.io/gitea/models/git=44.19%
code.gitea.io/gitea/models/issues=57.31%
code.gitea.io/gitea/models/organization=71.00%
code.gitea.io/gitea/models/packages=71.66%
code.gitea.io/gitea/models/perm=57.05%
code.gitea.io/gitea/models/project=31.85%
code.gitea.io/gitea/models/pull=36.13%
code.gitea.io/gitea/models/repo=58.62%
code.gitea.io/gitea/models/secret=24.56%
code.gitea.io/gitea/models/system=56.16%
code.gitea.io/gitea/models/unit=48.72%
code.gitea.io/gitea/models/unittest=31.91%
code.gitea.io/gitea/models/user=55.82%
code.gitea.io/gitea/models/webhook=66.02%
code.gitea.io/gitea/routers/api=60.66%
code.gitea.io/gitea/routers/common=61.15%
code.gitea.io/gitea/routers/install=7.39%
code.gitea.io/gitea/routers/private=34.74%
code.gitea.io/gitea/routers/utils=100.00%
code.gitea.io/gitea/routers/web=28.95%
code.gitea.io/gitea/services/agit=56.59%
code.gitea.io/gitea/services/asymkey=36.73%
code.gitea.io/gitea/services/attachment=60.87%
code.gitea.io/gitea/services/auth=38.99%
code.gitea.io/gitea/services/automerge=44.83%
code.gitea.io/gitea/services/context=47.22%
code.gitea.io/gitea/services/convert=74.80%
code.gitea.io/gitea/services/cron=61.44%
code.gitea.io/gitea/services/externalaccount=0.00%
code.gitea.io/gitea/services/forms=36.81%
code.gitea.io/gitea/services/gitdiff=70.52%
code.gitea.io/gitea/services/issue=40.50%
code.gitea.io/gitea/services/lfs=57.06%
code.gitea.io/gitea/services/mailer=54.55%
code.gitea.io/gitea/services/markup=100.00%
code.gitea.io/gitea/services/migrations=35.97%
code.gitea.io/gitea/services/mirror=29.83%
code.gitea.io/gitea/services/org=49.02%
code.gitea.io/gitea/services/packages=51.94%
code.gitea.io/gitea/services/pull=43.34%
code.gitea.io/gitea/services/release=45.09%
code.gitea.io/gitea/services/repository=41.19%
code.gitea.io/gitea/services/task=43.75%
code.gitea.io/gitea/services/user=46.24%
code.gitea.io/gitea/services/webhook=66.63%
code.gitea.io/gitea/services/wiki=56.41%
```